zebra/zebra_mpls.c

```diff
@@ -2638,8 +2638,16 @@ int zebra_mpls_write_fec_config(struct vty *vty, struct zebra_vrf *zvrf)
 				continue;
 
 			write = 1;
-			vty_out(vty, "mpls label bind %pFX %s\n", &rn->p,
-				label2str(fec->label, 0, lstr, BUFSIZ));
+
+			if (fec->label == MPLS_LABEL_IPV4_EXPLICIT_NULL ||
+			    fec->label == MPLS_LABEL_IPV6_EXPLICIT_NULL)
+				strlcpy(lstr, "explict-null", sizeof(lstr));
+			else if (fec->label == MPLS_LABEL_IMPLICIT_NULL)
+				strlcpy(lstr, "implict-null", sizeof(lstr));
```

Comment:
Typo

Reply:
Done, thanks!
Sorry, made typos in such a hurry.